src/ExpandArrayParameters.php

```diff
@@ -5,20 +5,23 @@
 use Doctrine\DBAL\ArrayParameters\Exception\MissingNamedParameter;
 use Doctrine\DBAL\ArrayParameters\Exception\MissingPositionalParameter;
 use Doctrine\DBAL\SQL\Parser\Visitor;
-use Doctrine\DBAL\Types\Type;
 
 use function array_fill;
 use function array_key_exists;
 use function count;
 use function implode;
 use function substr;
 
+/**
+ * @psalm-import-type ParameterTypeOfValue from ParameterType
+ * @psalm-type ArrayParameterTypeOfValue Connection::PARAM_*|ParameterTypeOfValue
+ */
```

Comment:
This will conflict with changes we've made on the 3.6.x branch already. And tbh, with everything you've changed now, I think we'd better not merge this as a bugfix and target 3.6.x instead.

Reply:
I will change it when rebase into 3.6.x